orchestration/dap_orchestration/solids.py

```diff
@@ -144,10 +166,13 @@ def env_transform_records(config: dict[str, str]) -> dict[str, str]:
         "working_dir": String,
     }
 )
-def write_outfiles(context: AbstractComputeExecutionContext) -> None:
+def write_outfiles(context: AbstractComputeExecutionContext, fan_in_results: list[object]) -> None:
```

Comment:
Even though it is ignored, the `fan_in_results` input is necessary to express the upstream dependencies.